### DISCUSSION TITLE
Why xaringan / remark.js for HTML5 Presentations? | /en/2017/08/why-xaringan-remark-js/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#955] [Comments]
https://yihui.org/en/2017/08/why-xaringan-remark-js/

[a comment by an automated account is not shown]

**Comment by ppreshant** [comment]
The link for [Compose Slides in R with Xaringan Slides by John R Little](https://www.johnlittle.info/post/compose-slides-in-r-with-xaringan-slides/) was broken, I put the correct one here

**Reply by yihui** [reply]
Thank you very much! I updated the link locally and will push to Github next time.
